ams/server/src/main/java/com/netease/arctic/server/ArcticManagementConf.java

```diff
@@ -224,6 +231,24 @@ public class ArcticManagementConf {
           .defaultValue("")
           .withDescription("The password for connecting to the database.");
 
+  public static final ConfigOption<Integer> DB_CONNECT_MAX_TOTAL =
+      ConfigOptions.key("database.connection-pool-max-total")
+          .intType()
+          .defaultValue(20)
+          .withDescription("Max connect count of database.");
+
+  public static final ConfigOption<Integer> DB_CONNECT_MAX_IDLE =
+      ConfigOptions.key("database.connection-pool-max-idle")
+          .intType()
+          .defaultValue(16)
+          .withDescription("Max idle connect count of database.");
```

Comment:
The term "description" can be more precise. It describes the maximum number of connections in the connection pool.